ci-operator/config/rhobs/observability-operator/rhobs-observability-operator-main.yaml

```diff
@@ -0,0 +1,72 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  operator-sdk:
+    name: "4.13"
+    namespace: origin
+    tag: operator-sdk
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- context_dir: .
+  dockerfile_path: build/Dockerfile
+  to: observability-operator
+- context_dir: .
+  dockerfile_path: test/Dockerfile
+  to: observability-operator-test-harness
+operator:
+  bundles:
+  - as: observability-bundle
+    context_dir: .
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/rhobs/observability-operator:0.0.25
+    with: pipeline:observability-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: observability-operator-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.13"
+  steps:
+    test:
+    - as: install
+      cli: latest
+      commands: |
+        oc create namespace obo
+        operator-sdk run bundle -n obo "$OO_BUNDLE" --install-mode AllNamespaces --skip-tls
```

Comment:
`--install-mode` AllNamespaces doesn't seem to work fine on ProwCI. Simple `operator-sdk run bundle -n obo "$OO_BUNDLE"` will be sufficient.